tox.ini

```diff
@@ -596,8 +598,7 @@ basepython=python
 
 # DEV: We use `conftest.py` as a local pytest plugin to configure hooks for collection
 [pytest]
-# Common directories to ignore
-addopts = --ignore "tests/utils" --ignore "tests/base" --durations=10
+addopts = --cov=ddtrace/ --cov-append --cov-report= --durations=10
```

Comment:
do these get added to subprocess invocations as well?

Reply:
They should yes https://pytest-cov.readthedocs.io/en/latest/subprocess-support.html#subprocess-support